factory/ethElrondBridgeComponents.go

```diff
@@ -310,7 +310,7 @@ func (components *ethElrondBridgeComponents) createEthereumClient(args ArgsEther
 		Broadcaster:               components.broadcaster,
 		PrivateKey:                privateKey,
 		TokensMapper:              tokensMapper,
-		SignatureHolder:           &disabledSignatureHolder{}, // TODO replace this with the real component
```

Comment:
this instance should be passed to the broadcaster instance as the broadcaster handles the new messages flow.